lfucg-exactions/server/accounts/auth.py

```diff
@@ -97,11 +99,11 @@ def post(self, request, *args, **kwargs):
 @api_view(['POST'])
 @permission_classes((AllowAny, ))
 def forgot_password(request):
-    email = request.data.get('email', None)
-    user = User.objects.filter(email__icontains=email)
-    if user.exists():
-        send_password_reset_email(user)
-        
+    email = request.data.get('email_forgot_password', None)
+    user = User.objects.get(email=email)
```

Comment:
Check first to make sure email is not None

Reply:
What happens if there is more than one user with that email?

Reply:
Do we still have to check if it is not none if that's what is being sent from the front end with messaging that says "if there is an email that matches we'll send you instructions"?

Also didn't test more than one person with that email. I can put in a safeguard for that... Are emails unique?

Reply:
Yes, we still need to test for if it is None.  Many checks like that are to prevent the system or users misbehaving.  
Emails are likely unique, but we are not putting any unique email requirements in the system, so there could be a repeat.